template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.gitignore

```diff
@@ -1,7 +1,7 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 ##
-## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
+## Get latest from `dotnet new gitignore`
```

Comment:
@baronfel do you have thoughts about it?

Reply:
Ideally they all would be the same source, but I'm comfortable leaning on `dotnet new` here. This does put more pressure on us to do something with https://github.com/dotnet/templating/issues/4126, but I think the net result is better for our users.